script.js

```diff
@@ -0,0 +1,108 @@
+"use strict";
+/*
+//1. Given an array consisting of movie names, iterate over the array with the output of the names of each .movie to the console.
+const movieNames = ["one","two","three","four","five","six","seven","eight","nine","ten"];
+for (let i =0; i<movieNames.length; i++) {
+    console.log(movieNames[i]);
+};
+*/
+
+/*
+//2.Given an array of car manufacturers, convert the array to a string and back to an array
+let carManufac = ["Toyota", "Ford", "Honda"];
+console.log(carManufac.join().split());
+*/
+
+/*
+//3.Given an array of the names of your friends, add the words hello to each element of the array
+let names = ["Michael", "Ares", "Henk"];
+for(let i =0 ;i < names.length ;i++) {
+    names[i] = "Hello " + names[i]; 
+}
+console.log(names);
+*/
+
+
+/*
+//4.Convert numeric array to Boolean
+let numArr = [1,6,8,0,-2].map(num =>Boolean(num));
+console.log(numArr);
+*/
+
+/*
+//5.Sort the array [1,6,7,8,3,4,5,6] in descending order
+let array =[1,6,7,8,3,4,5,6].sort((a,b)=> {
+    if (a<b) return 1;
+    if (a==b) return 0;
+    if (a>b) return -1;
+});
+console.log(array);
+*/
+
+/*
+//6.Filter array [1,6,7,8,3,4,5,6] by value > 3
+let arr = [1,6,7,8,3,4,5,6].filter((num)=>{
+    if (num > 3) return true
+});
+console.log(arr);
+*/
+
+/*
+//7.Write a function that takes two parameters - an array and a number and outputs the index of an array element equal to a number
+//If I understand you correctly..
```

Comment:
for exmple if you have an array [0, 1, 2, 3, 5], and you pass 5 in your function, it should return index of element that equal 5, so it should be 4